README.md

```diff
@@ -167,6 +167,7 @@ If you provided `PREFIX` and/or `DESTDIR` while running `make install`, remember
 | `make hosts-firefox-user`  | Configure browserpass for Firefox browser, for the current user only                 |
 | `make hosts-librewolf-user`| Configure browserpass for Librewolf browser, for the current user only               |
 | `make hosts-chrome-user`   | Configure browserpass for Google Chrome or Opera browsers, for the current user only |
+| `make hosts-edge-user`     | Configure browserpass for Microsoft Edge browser, for the current user only          |
```

Comment:
In line with the suggestion above, let's add the `hosts-edge` to the README as well 😉

Reply:
Change pushed!